updater/reports/ReportRepositoryHistory.py

```diff
@@ -0,0 +1,52 @@
+from .ReportDaily import *
+
+# Lists the total number of repositories
+class ReportRepositoryHistory(ReportDaily):
+	def name(self):
+		return "repository-history"
+
+	def updateDailyData(self):
+		print(self.query())
+	
+		newHeader, newData = self.parseData(
+			self.executeQuery(self.query())
+		)
+		self.header = newHeader if newHeader else self.header
+		self.data.extend(newData)
+		self.truncateData(self.timeRangeTotal())
+		self.sortDataByDate()
+
+	# Collects the number of active repositories for a user type (user/organization) given time range
+	def subquery(self, userType):
+		query = '''
+			SELECT
+				COUNT(*) AS count
+			FROM
+				repositories
+				JOIN users ON repositories.owner_id = users.id
+			WHERE
+				TRUE'''
```

Comment:
TRUE?

Reply:
Got it!

Reply:
For the record, this was done to handle the special case where there are no excluded entities *and* `users.type == None`.

In this case, there wouldn’t be any content to the `WHERE` statement, causing a syntax error.